pkg/storage/tsdb/expanded_postings_cache.go

```diff
@@ -140,6 +141,11 @@ type blocksPostingsForMatchersCache struct {
 	seedByHash *seedByHash
 }
 
+func (c *blocksPostingsForMatchersCache) Clear() {
```

Comment:
Is this only used in tests?

Reply:
for now yeah.. u wanna me to add a comment there?